web/src/components/Sidebar.js

```diff
@@ -28,10 +28,7 @@ export default function Sidebar({ currentProject, setCurrentProject, setIsEditin
         </SectionList>
         <UserDetails
           user={app.currentUser}
-          handleLogout={async () => {
-            await app.currentUser.logOut();
-            refresh();
-          }}
+          handleLogout={() => { app.logOut() }}
```

Comment:
```suggestion
          handleLogout={() => {logOut() }}
```

see: https://github.com/mongodb-university/realm-tutorial/pull/69/commits/3818f3dade6b5353eb5515fcbebac4868c3e2e2d#r492868658